Lombiq.HelpfulLibraries.Common/Utilities/FileSystemHelper.cs

```diff
@@ -0,0 +1,27 @@
+using System.IO;
+using System.Linq;
+
+namespace Lombiq.HelpfulLibraries.Common.Utilities;
+
+/// <summary>
+/// Some shortcuts for file system operations.
+/// </summary>
+public static class FileSystemHelper
+{
+    /// <summary>
+    /// Returns a composed file system path from the <paramref name="pathComponents"/> and creates it as a directory if
+    /// it doesn't exist.
+    /// </summary>
+    public static string EnsureDirectoryExists(params string[] pathComponents)
+    {
+        var path = Path.Combine(pathComponents);
+        if (!Directory.Exists(path)) Directory.CreateDirectory(path);

```

Comment:
Can't this throw if a file with the given path already exists?

Reply:
It will and it should:
```
Cannot create '{path}' because a file or directory with the same name already exists.
```
I think this is perfectly descriptive and enough information for the user to fix their own mess, so we don't need to do anything.

Reply:
Makes perfect sense.